src/features/pagination/js/pagination.js

```diff
@@ -368,8 +410,6 @@
           $scope.$on('$destroy', dataChangeDereg);
 
           var setShowing = function () {
```

Comment:
It looks like this is an empty function now. Was that on purpose? If so, what is the intention here?